ecs-cli/modules/cli/local/docker/docker.go

```diff
@@ -21,9 +21,13 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+// Wait durations for a response from the Docker daemon before returning an error to the caller.
 const (
-	// TimeoutInS is the wait duration for a response from the Docker daemon before returning an error to the user.
+	// TimeoutInS is the wait duration for common operations.
 	TimeoutInS = 30 * time.Second
+
+	// LongTimeoutInS is the wait duration for long operations such as PullImage.
+	LongTimeoutInS = 120 * time.Second
```

Comment:
How would we use the status of the image pulls? Is it to write a status update to the customer or did you have something else in mind?

Reply:
This is cool! Thanks for sharing :) 
I agree that's a bit too bulky, could we instead use [context.WithCancel](https://golang.org/pkg/context/#WithCancel) to achieve the same thing?

The image is 17.4MB so with 2 mins they would need at least 152KB/s which sounds reasonable to me? We could also just increase the timeout to something like 5 mins instead.

Reply:
Hmm, I'm not sure whether it will be easy to renew the parent context with a child context created using `context.WithCancel`. 

Agree that this level of complexity is not necessary now :) since most customers will have 1.2 Mbps ha